### PR TITLE
 Redis session store implemented to enable  horizontal scaling 

### PR DESCRIPTION
**Pull request by sagargg**
**Fixes for:** #188
- Redis session store implemented. 
- Added `REDIS_URL` env variable. 
- When there is no `REDIS_URL` value,  by default the session will be stored on the app memory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ Description of each variable:
 * `NODE_ENV` - this is just a node environment to indicate in which environment you are. If it is unset or `development`, the app will use mocked APIs. If you're working against live endpoint, set this config to, e.g., `staging`.
 * `THEME` - name of your theme.
 * `PLUGINS` - list of space separated plugins, e.g., if you're using wordpress and dashboard: `wp dashboard`.
+* `REDIS_URL` - Provide redis url for the production envionment inorder store session related data on Redis database.
```
Comment (anuveyatsu) [inline README.md:76]: @sagargg could you fix this sentence?

Comment (sagargg) [reply README.md]: @anuveyatsu  I've updated the sentence.

File: index.js [hunk 2]
```diff
@@ -63,17 +65,49 @@ module.exports.makeApp = function () {
...
+  // Handling session error
+  app.use(function (req, res, next) {
+    if (!req.session) {
+      var err = new Error();
+      err.status = 500;
+      err.statusText = 'SESSION EXPIRED OR NOT FOUND'
+      err.text = () => {
+        return 'Failed to configure session.'
+      }
+      next(err)
```
Comment (anuveyatsu) [inline index.js:106]: @sagargg does that mean the app will through 500 if there is no session?

Comment (anuveyatsu) [reply index.js:106]: @sagargg can we update status text to something more user-friendly, eg:

> Something has failed. Please, try again later.

Comment (sagargg) [reply index.js:106]: @anuveyatsu  updated.

**Review by anuveyatsu — APPROVED** [review]
:+1:

**Pull request merged**